src/ui/components/Library/RecordingRow.tsx

```diff
@@ -139,6 +145,13 @@ function RecordingRow({
                 <div className="overflow-hidden overflow-ellipsis whitespace-pre font-light text-gray-400">
                   {getDisplayedUrl(recording.url)}
                 </div>
+                {
+                  recording.metadata?.test ? (
```

Comment:
This probably won't affect layout but it might be worth checking if `file` is set here so we don't add a node that is empty.

e.g. `recording.metadata?.test?.file`